frontend/app/src/components/home/ChatEvent.svelte

```diff
@@ -61,6 +62,9 @@
 
     let userSummary: UserSummary | undefined = undefined;
 
+    $: level = $_(`level.${chatType === "channel" ? "channel" : "group"}`).toLowerCase() as
```

Comment:
This seems weird... Its grabbing the translation of 'channel' or 'group' then casting it to type "channel" | "group" which will be wrong for all languages other than English.